src/components/pages/Cases/CaseTable.js

```diff
@@ -164,13 +164,76 @@ export default function CaseTable(props) {
     return month + '/' + day + '/' + year;
   }
 
-  function changeSorter(sorter) {
-    console.log(sorter);
+  // state to keep track of filters being applied to the table (Initial cases section)
+  const [initialFilters, setInitialFilters] = useState([]);
+  // keeping track of filters applied to the appelate section
+  const [appFilters, setAppFilters] = useState([]);
+
+  // state to keep track of the current table being displayed
+  const [currentKey, setCurrentKey] = useState(1);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(initialFilters);
+  }, [initialFilters]);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(appFilters);
+  }, [appFilters]);
+
+  // returns processed array of filters
+  const processFilters = filters => {
```

Comment:
I can see this looking clean with a switch statement

Reply:
Hmm.. I think a switch statement in this case would convolute things even more. Looping through an object with a lot of keys, many of which are null keys, is the best way I could thing to implement this ```processFilters``` function. I'll take it into consideration though.